### PR TITLE
chore: Test Docusaurus webpack 5 support

### PR DESCRIPTION
**Pull request by slorber**
Use an unreleased version of Docusaurus to test the webpack5 support works for this site.

See also https://github.com/facebook/docusaurus/pull/4089

### CODE REVIEW

**Comment by juzhiyuan** [comment]
Wow! Welcome & Thanks for your PR!

**Comment by slorber** [comment]
Thanks 😉 happy to see i18n used there👌

And thanks for the sponsorship

**Comment by slorber** [comment]
ready to review, let me know if anything does not work

File: website/src/theme/DocSidebar/index.js [hunk 1]
```diff
@@ -162,19 +175,14 @@ function DocSidebar({
...
       hideOnScroll
     },
     hideableSidebar
   } = useThemeConfig();
-  const {
-    isAnnouncementBarClosed
-  } = useUserPreferencesContext();
-  const {
-    scrollY
-  } = useScrollPosition();
```
Comment (slorber) [inline website/src/theme/DocSidebar/index.js:177]: breaking change on the useScrollPosition internal hook

**Review by juzhiyuan — APPROVED** [review]
LGTM from my side

**Review by bzp2010 — APPROVED** [review]

**Pull request merged**